uwblueprint/sistering
# Admin create posting "Review and Post" page UI

## Ticket link

Closes #59 



## Implementation description
* Created the posting review component `CreatePostingReview.tsx` (under `frontend/src/components/admin/posting`) and the review page `CreatePostingReviewPage.tsx` (under `frontend/src/components/pages/admin/posting`) in which the component is mounted
* Routed the page to `/admin/posting/create/review`
* Used `PostingContext` to fetch posting data to be displayed in the review page
* Added following components to `frontend/src/components/common`:
  * `BasicInfoGridItem.tsx`
  * `PocCard.tsx`
  * `SkillTag.tsx`
  * `ScheduledShiftsTr.tsx`


## Steps to test
1. Use the Firebase preview URL below or run the application locally `docker-compose up --build`
2. Log in and click on "Edit Posting" tab on the default page and enter some dummy information in the form
3. Go back to the default page and click on "Review Posting" tab
4. Verify the changes on the review posting page. Try jumping back and forth between these pages and manipulating the data. Note that you should not reload the app as that will clear the app state. This means you should not refresh or navigate by manually changing the URL, instead, navigate using the tabs and "Go Back" buttons)



## What should reviewers focus on?
* Does `CreatePostingReview.tsx` follow the Figma design?
* Is the page properly showing the posting info (from context)?


## Checklist
